packages/flutter/test/widgets/framework_test.dart

```diff
@@ -486,24 +486,23 @@ void main() {
     expect(
       element.toStringDeep(),
       equalsIgnoringHashCodes(
-        '''Column([GlobalKey#00000]; renderObject: RenderFlex#00000)
-├Container()
-│└LimitedBox(maxWidth: 0.0; maxHeight: 0.0; renderObject: RenderLimitedBox#00000 relayoutBoundary=up1)
-│ └ConstrainedBox(BoxConstraints(biggest); renderObject: RenderConstrainedBox#00000 relayoutBoundary=up2)
-├Container([GlobalKey#00000])
-│└LimitedBox(maxWidth: 0.0; maxHeight: 0.0; renderObject: RenderLimitedBox#00000 relayoutBoundary=up1)
-│ └ConstrainedBox(BoxConstraints(biggest); renderObject: RenderConstrainedBox#00000 relayoutBoundary=up2)
-├Container()
-│└Container()
-│ └LimitedBox(maxWidth: 0.0; maxHeight: 0.0; renderObject: RenderLimitedBox#00000 relayoutBoundary=up1)
-│  └ConstrainedBox(BoxConstraints(biggest); renderObject: RenderConstrainedBox#00000 relayoutBoundary=up2)
-├Container([GlobalKey#00000])
-│└LimitedBox(maxWidth: 0.0; maxHeight: 0.0; renderObject: RenderLimitedBox#00000 relayoutBoundary=up1)
-│ └ConstrainedBox(BoxConstraints(biggest); renderObject: RenderConstrainedBox#00000 relayoutBoundary=up2)
-└Container()
- └LimitedBox(maxWidth: 0.0; maxHeight: 0.0; renderObject: RenderLimitedBox#00000 relayoutBoundary=up1)
-  └ConstrainedBox(BoxConstraints(biggest); renderObject: RenderConstrainedBox#00000 relayoutBoundary=up2)
-''',
+        'Column([GlobalKey#00000]; renderObject: RenderFlex#00000)\n'
```

Comment:
Change here is Container
is displayed instead of
Container()
I can make the Dense style put parens around empty parameter lists but that seems odd.
I am assuming placing parens around empty param lists was just an oversight as the similar existing singleLine style did not place parens around empty parameter lists. Just let me know. It is trivial to switch this default back.